gobreaker.go

```diff
@@ -18,6 +19,27 @@ const (
 	StateOpen
 )
 
+// ErrTooManyRequest is returned when the CB state is half open and the requests count is over the cb maxRequests
+var ErrTooManyRequest = errors.New("too many requests")
```

Comment:
Did you mean to name this `ErrTooManyRequests` or `ErrTooManyRequest`?

Reply:
Gotcha! :+1: 
Renamed to `ErrTooManyRequests` and added a test for the complaining coverage.